__tests__/convertColor.test.ts

```diff
@@ -0,0 +1,52 @@
+import { 
+	figmaRGBToWebRGB,
+	webRGBToFigmaRGB,
+	figmaRGBToHex,
+	hexToFigmaRGB
+ } from '../src'
+
+
+const figmaRGB = { r: 0, g: 0.1, b: 1 }
+const figmaRGBA = { r: 0, g: 0.1, b: 1, a: .5 }
+
+const webRGB: [number, number, number] = [0, 26, 255]
+const webRGBA: [number, number, number, number] = [0, 26, 255, .5]
+
+const HEX = "#001aff"
+const HEXa = "#001aff80"
+
+describe('convertColor', () => {
+	test('figmaRGB to webRGB', () => {
+		expect(figmaRGBToWebRGB(figmaRGB)).toEqual(webRGB)
+	})
+	test('figmaRGBA to webRGBA', () => {
+		expect(figmaRGBToWebRGB(figmaRGBA)).toEqual(webRGBA)
+	})
+	test('webRGB to figmaRGB', () => {
+		expect(webRGBToFigmaRGB(webRGB)).toEqual({ r: 0, g: 0.10196078431372549, b: 1 })
+	})
+	test('webRGBA to figmaRGBA', () => {
+		expect(webRGBToFigmaRGB(webRGBA)).toEqual({ r: 0, g: 0.10196078431372549, b: 1, a: 0.5 })
```

Comment:
Why `g` value has so many symbols after dot? Does it's expected? I thought it should be equal to `figmaRGB` constant, it seems like an issue with `webRGBToFigmaRGB()`

Reply:
I think it's okay to add rounding for this values. Issue from screenshot is related to JS lang maths: 0.001 + 0.002 !== 0.003

Reply:
> I think it's okay to add rounding for this values

By adding it, we will lose accuracy again and the reverse conversion will no longer be the same: `figmaRGBToWebRGB({ r: 0, g: 0.10196078431372549, b: 1 }) === webRGB`

Figma doesn't round!

> Issue from screenshot is related to JS lang maths

Hmm ... exactly.

Reply:
@alexandrtovmach .

Reply:
1. I think that's better to update this helper by adding rounding, but it's separate task
2. Define all _expected_ values as variables at the start of tests